src/dmd/dinterpret.d

```diff
@@ -1089,11 +1089,11 @@ public:
         if (istate.start == s)
             istate.start = null;
 
-        size_t dim = s.statements ? s.statements.dim : 0;
-        for (size_t i = 0; i < dim; i++)
+        const dim = s.statements ? s.statements.dim : 0;
+        foreach (i; 0 .. dim)
```

Comment:
Please use a regular foreach loop instead of a range foreach if possible.

Reply:
I didn't because look at the declaration for `dim`, there's a null check that wouldn't be there in the foreach.

Reply:
Make sense.